### PR TITLE
feat(device): add smart plug compact

### PR DESCRIPTION
**Pull request by Purii**
Add support for Smart Plug Compact with ZigBee 3.0.

Output of `SHCDevice.summary`

```
PLUG_COMPACT SmartPlugCompact:
Device: hdm:ZigBee:12345
  Name          : Some Name
  Manufacturer  : BOSCH
  Model         : PLUG_COMPACT
  Room          : hz_6
  Serial        : 847127FFFE99DB9B
  Profile       : GENERIC
  Status        : AVAILABLE
  ParentDevice  : None
  ChildDevices  : []
  Device Service: CommunicationQuality
    State: {'@type': 'communicationQualityState', 'quality': 'GOOD'}
    Path:  /devices/hdm:ZigBee:12345/services/CommunicationQuality
    quality                  : State.GOOD
  Device Service: PowerMeter
    State: {'@type': 'powerMeterState', 'powerConsumption': 0.0, 'energyConsumption': 0.0}
    Path:  /devices/hdm:ZigBee:12345/services/PowerMeter
    powerConsumption         : 0.0
    energyConsumption        : 0.0
  Device Service: PowerSwitch
    State: {'@type': 'powerSwitchState', 'switchState': 'OFF', 'automaticPowerOffTime': 0}
    Path:  /devices/hdm:ZigBee:12345/services/PowerSwitch
    switchState              : State.OFF
    automaticPowerOffTime    : 0
  Device Service: PowerSwitchProgram
    State: {'@type': 'powerSwitchProgramState', 'operationMode': 'MANUAL', 'schedule': {'profiles': []}}
    Path:  /devices/hdm:ZigBee: 12345/services/PowerSwitchProgram
    operationMode            : State.MANUAL
```

### CODE REVIEW

File: boschshcpy/models_impl.py [hunk 1]
```diff
@@ -117,6 +117,55 @@ def summary(self):
...
+    @property
+    def energyconsumption(self) -> float:
+        return self._powermeter_service.energyconsumption
+
+    @property
+    def powerconsumption(self) -> float:
+        return self._powermeter_service.powerconsumption
+
+    @property
+    def communicationquality(self) -> float:
```
Comment (tschamm) [inline boschshcpy/models_impl.py:155]: Please substitute return type `float` by  `CommunicationQualityService.State`.

**Review by tschamm — CHANGES_REQUESTED** [review]
@Purii Thanks for your contribution. Everything looks pretty fine, except the return type. Can you fix this, so that I can merge your commit.

File: boschshcpy/models_impl.py (hunk 1 above)
Comment (Purii) [reply boschshcpy/models_impl.py]: Thanks for reviewing. Just fixed it!

**Review by tschamm — APPROVED** [review]
Changes look fine. Thank you very much!

**Pull request merged**